entity-service/src/test/kotlin/com/egm/stellio/entity/service/EntityOperationServiceTests.kt

```diff
@@ -51,14 +51,28 @@ class EntityOperationServiceTests {
         val secondEntity = mockkClass(NgsiLdEntity::class)
         every { secondEntity.id } returns "2"
 
-        every { neo4jRepository.filterExistingEntitiesIds(listOf("1", "2")) } returns listOf("1")
+        every { neo4jRepository.filterExistingEntitiesAsIds(listOf("1", "2")) } returns listOf("1")
 
         val (exist, doNotExist) = entityOperationService.splitEntitiesByExistence(listOf(firstEntity, secondEntity))
 
         assertEquals(listOf(firstEntity), exist)
         assertEquals(listOf(secondEntity), doNotExist)
     }
 
+    @Test
+    fun `it should split entities per existence with ids`() {
+        val firstEntity = "1"
```

Comment:
Can you use meaningful names ? 
for instance: 

- urn:ngsi-ld:Device:HCMR-AQUABOX1
- urn:ngsi-ld:Device:HCMR-AQUABOX2

Reply:
OK